src/exceptions/LLGPL.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="LLGPL"
+   name="LLGPL Preamble" listVersionAdded="3.21">
+      <crossRefs>
+         <crossRef>http://opensource.franz.com/preamble.html</crossRef>
+      </crossRefs>
+      <notes>
+         Typically used with LGPL-2.1
+      </notes>
+      <text>
+         <titleText>Preamble to the Gnu Lesser General Public License
+         </titleText>
+         <copyrightText>
+            Copyright (c) 2000 Franz Incorporated, Berkeley, CA 94704
+         </copyrightText>
+         <p>
+            The concept of the GNU Lesser General Public License version 2.1
+            ("LGPL") has been adopted to govern the use and distribution of
+            <alt match="above-mentioned application|AllegroServe" name="app1">above-mentioned application</alt>. 
+            However, the LGPL uses terminology
+            that is more appropriate for a program written in C than one
+            written in Lisp. Nevertheless, the LGPL can still be applied to
+            a Lisp program if certain clarifications are made. This document
+            details those clarifications. Accordingly, the license for
+            <alt match="the open-source Lisp applications|AllegroServe" name="app">the
+            open-source Lisp applications</a> consists of this document plus

```

Comment:
@jlovejoy I think this is failing because the opening tag is `<alt>` but the closing tag is `</a>`. I'm going to fix it and see if that resolves this one.